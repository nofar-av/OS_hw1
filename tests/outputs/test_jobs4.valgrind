==28285== Memcheck, a memory error detector
==28285== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28285== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28285== Command: /home/omer/OS_wet1/smash
==28285== Parent PID: 28280
==28285== 
==28285== 
==28285== FILE DESCRIPTORS: 4 open at exit.
==28285== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_jobs4.valgrind
==28285==    <inherited from parent>
==28285== 
==28285== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_jobs4.err
==28285==    <inherited from parent>
==28285== 
==28285== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_jobs4.out
==28285==    <inherited from parent>
==28285== 
==28285== Open file descriptor 0:
==28285==    <inherited from parent>
==28285== 
==28285== 
==28285== HEAP SUMMARY:
==28285==     in use at exit: 0 bytes in 0 blocks
==28285==   total heap usage: 33 allocs, 33 frees, 82,432 bytes allocated
==28285== 
==28285== All heap blocks were freed -- no leaks are possible
==28285== 
==28285== For counts of detected and suppressed errors, rerun with: -v
==28285== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
