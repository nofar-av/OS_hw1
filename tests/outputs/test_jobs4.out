smash> smash> smash> smash> smash: got ctrl-Z
